### ISSUE TITLE
Update dependencies for target `net6.0`

### ISSUE THREAD

**Issue by Eagle3386** [mheyman/Isopoh.Cryptography.Argon2#43]
Hey, @mheyman!

I just found your repo & tried to install it into my .NET 6-based project.
However, your dependencies for target `net6.0` explicitly list `System.Thread`, `System.ThreadPool` & `System.ValueTuple`, yet that very target already includes them on its own.

Would you therefore mind updating your dependencies regarding that platform?
I'd happily provide you with a PR, but couldn't find the required `.nuspec` file in your repo...

**Comment by mheyman** [comment]
Doh! those were unused package references that I left in there unknowingly. I know how frustrating that kind of fail can be to come across. 1.1.12 uploaded with *no* unused package references. :wink:

**Comment by mheyman** [comment]
Well, I'm not sure if this is a regression or not.

I'll tell you what I did. I ran "remove unused references" in Visual Studio on all the projects. It got rid of `System.Memory`, `System.Runtime`, and `System.Runtime.Extensions`. These now get pulled in from the actual dependency of `netstandard2.0`. I don't know what pulls in `Microsoft.NETCore...` (at least with my quick 5-minute hunt through the solution).

Obviously, the code works with the older dependencies or the newer ones if you want to explicitly pull them in with your project.
